### PR TITLE
Move to new ci

### PR DESCRIPTION
**Pull request by mgrassotti**
Trello: https://trello.com/c/w1nBZRNQ/579-move-release-to-new-ci-infrastructure-1

The old CI infrastructure is in the process of being wound down so we need to move our apps to the new one.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'release'
+DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
```
Comment (steventux) [inline Jenkinsfile]: [It doesn't look like this app includes schema tests](https://github.com/alphagov/release/blob/master/jenkins.sh) so this and other schema related config isn't necessary.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          description: 'Identifies whether this build is being triggered to test a change to the content schemas'],
+        [$class: 'StringParameterDefinition',
+          name: 'SCHEMA_BRANCH',
+          defaultValue: DEFAULT_SCHEMA_BRANCH,
+          description: 'The branch of govuk-content-schemas to test against']]
+    ],
+  ])
+
+  try {
+    govuk.initializeParameters([
```
Comment (steventux) [inline Jenkinsfile]: Lines 37-45 are not needed as this app doesn't include schema tests.

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+    stage("git merge") {
+      govuk.mergeMasterBranch()
+    }
+
+    stage("Configure Rails environment") {
+      govuk.setEnvar("RAILS_ENV", "test")
+    }
+
+    stage("Set up content schema dependency") {
```
Comment (steventux) [inline Jenkinsfile]: Not necessary as mentioned.

Comment (mgrassotti) [inline Jenkinsfile]: 👍

File: Jenkinsfile [hunk 4]
```diff
@@ -19,30 +18,11 @@ node {
       maxConcurrentTotal: 0,
       paramsToUseForLimit: 'release',
       throttleEnabled: true,
```
Comment (steventux) [inline Jenkinsfile]: There's no throttling on the old configuration https://ci.dev.publishing.service.gov.uk/job/govuk_release/configure

File: Jenkinsfile [hunk 5]
```diff
@@ -19,30 +18,11 @@ node {
...
-      parameterDefinitions: [
-        [$class: 'BooleanParameterDefinition',
-          name: 'IS_SCHEMA_TEST',
-          defaultValue: false,
-          description: 'Identifies whether this build is being triggered to test a change to the content schemas'],
-        [$class: 'StringParameterDefinition',
-          name: 'SCHEMA_BRANCH',
-          defaultValue: DEFAULT_SCHEMA_BRANCH,
-          description: 'The branch of govuk-content-schemas to test against']]
+      throttleOption: 'category'
```
Comment (steventux) [inline Jenkinsfile]: This option isn't needed and if it were it usually has a parameter (for the category to throttle on)

Comment (mgrassotti) [inline Jenkinsfile]: 👍

**Pull request merged**